lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -528,6 +528,10 @@ module.exports = class MethodMappings {
 
           try {
             const element = this.getWebElement(webElement);
+
+            // clear Element value
+            await this.methods.session.clearElementValue.call(this, webElement);
```

Comment:
can we make it more simpler by duplicating

await element.clear();

Reply:
Yes, fixed 👍